_posts/2022-11-10-release-1.15.3.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+title:  "Apache Flink 1.15.3 Release Announcement"
+date: 2022-11-10T22:00:00.000Z
+categories: news
+authors:
+- fapaul:
+  name: "Fabian Paul"
+
+excerpt: The Apache Flink Community is pleased to announce a bug fix release for Flink 1.15.
+
+---
+
+The Apache Flink Community is pleased to announce the second bug fix release of the Flink 1.15 series.
+
+This release includes 59 bug fixes, vulnerability fixes, and minor improvements for Flink 1.15.
+Below you will find a list of all bugfixes and improvements (excluding improvements to the build infrastructure and build stability). For a complete list of all changes see:
+[JIRA](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12352210).
+
+We highly recommend all users upgrade to Flink 1.15.3.
+
+# Release Artifacts
+
+## Maven Dependencies
+
+```xml
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-java</artifactId>
+  <version>1.15.3</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-streaming-java</artifactId>
+  <version>1.15.3</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-clients</artifactId>
+  <version>1.15.3</version>
+</dependency>
+```
+
+## Binaries
+
+You can find the binaries on the updated [Downloads page]({{ site.baseurl }}/downloads.html).
+
+## Docker Images
+
+* [library/flink](https://hub.docker.com/_/flink?tab=tags&page=1&name=1.15.3) (official images)
+* [apache/flink](https://hub.docker.com/r/apache/flink/tags?page=1&name=1.15.3) (ASF repository)
+
+## PyPi
+
+* [apache-flink==1.15.3](https://pypi.org/project/apache-flink/1.15.3/)
+
+# Release Notes
+
+<h2>        Bug
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-25554'>FLINK-25554</a>] -         DistributedCacheDfsTest.testSubmittingJobViaRestClusterClient fails on AZP
```

Comment:
Remove this as per above: "(excluding improvements to the build infrastructure and build stability)"